src/Themes/VuetifyTheme/Migrations.cs

```diff
@@ -1274,5 +1280,83 @@ private void CreateFip()
                 )
             );
         }
+
+        private void VListTitle() {
+            _contentDefinitionManager.AlterTypeDefinition("VListTitle", type => type
+                .DisplayedAs("VListTitle")
+                .WithPart("VListTitle", part => part
+                    .WithPosition("0")
+                )
+            );
+
+            _contentDefinitionManager.AlterPartDefinition("VListTitle", part => part
+                .WithField("Title", field => field
+                    .OfType("TextField")
+                    .WithDisplayName("Title")
+                    .WithPosition("0")
+                )
+            );
+        }
+        private void VListItem() {
+            _contentDefinitionManager.AlterTypeDefinition("VListItem", type => type
+                .DisplayedAs("VListItem")
+                .WithPart("VListItem", part => part
+                    .WithPosition("0")
+                )
+            );
+
+            _contentDefinitionManager.AlterPartDefinition("VListItem", part => part
+                .WithTextField("ItemTitle", "Item Title", "0")
+                .WithTextField("IconName", "Icon Name", "1")
+                .WithTextField("ItemSubTitle", "Item Subtitle", "2")
+                .WithTextField("ItemText", "Item Text", "3")
+            );
+        }
+
+        private void VList() {
+            _contentDefinitionManager.AlterTypeDefinition("VList", type => type
+                .DisplayedAs("VList")
+                .Creatable()
+                .Listable()
+                .Draftable()
+                .Versionable()
+                .Securable()
+                .Stereotype("Widget")
+                .WithPart("VList", part => part
+                    .WithPosition("0")
+                )
+                .WithPart("Items", "BagPart", part => part
+                    .WithDisplayName("Items")
+                    .WithDescription("Provides a collection behavior for your content item where you can place other content items.")
+                    .WithPosition("1")
+                    .WithSettings(new BagPartSettings
+                    {
+                        ContainedContentTypes = new[] { "VDivider", "VListItem", "VListTitle" },
+                    })
+                )
+                
+            );
+        }
+        private void VDivider() {
+            _contentDefinitionManager.AlterTypeDefinition("VDivider", type => type
+                .DisplayedAs("VDivider")
+                .Stereotype("Widget")
+                .WithPart("VDivider", part => part
+                    .WithPosition("0")
+                )
+            );
+        }
+        public int UpdateFrom1() {
+            CompatibilityBanner();
```

Comment:
`CompatibilityBanner()`, `VFooter()`, and `UpdateToMultiTextField()` can be removed from `UpdateFrom1()`. The migration only needs to add the new content types.

Reply:
Don't forget to increate the version to 2 on the Create() migration as well if you are creating all the types there.